importers/package.json

```diff
@@ -33,11 +33,11 @@
         "@types/sax": "^1.2.0",
         "copyfiles": "^2.1.0",
         "jest": "24.5.0",
-        "sqlite3": "5.0.0",
+        "sqlite3": "5.0.2",
         "ts-jest": "24.1.0",
-        "ts-node": "9.0.0",
+        "ts-node": "9.1.1",
         "tslint": "6.1.3",
-        "typescript": "4.0.3",
+        "typescript": "4.2.3",
```

Comment:
lets upgrade to a more modern version of typescript

Reply:
Not sure what you mean, 4.2.3 is the newest version already

Reply:
ok, i guess you just meant: yay, lets do it! :-)